code/Model/Recipient/EmailRecipient.php

```diff
@@ -511,7 +511,7 @@ public function getEmailTemplateDropdownValues()
             $templatePath = substr($absoluteFilename, strlen($prefixToStrip) + 1);
 
             // Optionally remove "templates/" prefixes
-            if (preg_match('/(?<=templates\/).*$/', $templatePath, $matches)) {
+            if (preg_match('/(?<=templates(\/|\\\)).*$/', $templatePath, $matches)) {
```

Comment:
It looks like this should cause an error in PCRE compilation.
Could you please update the triple backslash to be just 2?
`(\/|\\)`